_posts/2018-10-01-asdf-version-management.md

```diff
@@ -0,0 +1,136 @@
+---
+author: Bobby Grayson
+author_link: https://github.com/notactuallypagemcconnell
+categories: general
+date: 2018-10-01
+layout: post
+title:  Agnostic Version Management With ASDF
+excerpt: >
+  Take a dive into flexible version management of Elixir, Erlang, and OTP with `asdf`!
+---
+
+# Agnostic Version Management With ASDF
```

Comment:
The name 'asdf' is all lowercase, like 'sed' and 'awk'. It is not an acronym.

Reply:
Got it. Thank you.